version1/02 - Object types.md

```diff
@@ -500,6 +501,55 @@ The following notation is used in the tables below:
+| M601      | string    | opprettetAvBrukerIdent           |    x     |      x      |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M602      | timestamp | avsluttetDato                    |          |             |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M603      | string    | avsluttetAv                      |          |             |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M603      | string    | avsluttetAvBrukerIdent           |          |             |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M004      | string    | registreringsIdent               |    x     |      x      |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M020      | string    | tittel                           |    x     |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M025      | string    | offentligTittel                  |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M021      | string    | beskrivelse                      |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+| M024      | string    | forfatter                        |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                     |
+|           | string    | publiseringsstatus               |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | Allowed values: NOT_PUBLISHED, PUBLISHED_METADATA, PUBLISHED_METADATA_AND_DOCUMENTS                                 |
```

Comment:
Should be NOT_PUBLISHED only.

Reply:
Fixed w/ https://github.com/documaster/noark5-web-services/pull/63/commits/c6f7a06a9a737238d92854491a6727e60d56680f